app/server/html.ts

```diff
@@ -27,13 +27,17 @@ const html: (_: {
       <meta charset="utf-8">
       <meta name="viewport" content="width=device-width, initial-scale=1">
       <title>${title}</title>
+
       ${insertGlobals(globals)}
+      <script src="https://js.stripe.com/v3/"></script>
```

Comment:
Does this mean Stripe is loaded on every page rather than just the card payment screen?

Reply:
good spot. i feel sure there's privacy requirement not to do this. This is why stripe is loading asynchronously using `loadStripe` function, only when it's needed https://github.com/guardian/manage-frontend/blob/e2efc7219fead3918a176efd1089cdf7b61899c9/app/client/stripe.ts#L33

Reply:
should be fine to load it on every page in cypress, but I'm curious why 

also as general rule one should try to keep changes to production code (particularly branching logic) for the sole purpose of testing - to a minimum

Reply:
thanks for the suggestion richard, works nicely